packages/widget-chat/src/components/typing-indicator/styles.css

```diff
@@ -0,0 +1,10 @@
+.background {
+  width: 30px;
+  height: 30px;
+  padding: 0;
+  text-align: center;
+  cursor: default;
+  background-color: #696969;
+  border: none;
+  border-radius: 15px;
+}
```

Comment:
Does the current web client UI animate the ellipses? If so, we may want to get that in now.